lib/friendly_shipping/services/ups/shipping_methods.rb

```diff
@@ -0,0 +1,102 @@
+        ['EU', 'domestic', 'UPS Access Point Economy Shipments within the European Union', '70'],
+        ['EU', 'international', 'UPS Expedited Shipments originating in the European Union', '08'],
+        ['EU', 'international', 'UPS Express Shipments originating in the European Union', '07'],
+        ['EU', 'international', 'UPS Standard Shipments originating in the European Union', '11'],
+        ['EU', 'international', 'UPS Worldwide Express Plus Shipments originating in the European Union', '54'],
+        ['EU', 'international', 'UPS Worldwide Saver Shipments originating in the European Union', '65'],
+        ['MX', 'domestic', 'UPS Access Point Economy Shipments within Mexico', '70'],
+        ['MX', 'international', 'UPS Expedited Shipments originating in Mexico', '08'],
+        ['MX', 'international', 'UPS Express Shipments originating in Mexico', '07'],
+        ['MX', 'international', 'UPS Standard Shipments originating in Mexico', '11'],
+        ['MX', 'international', 'UPS Worldwide Express Plus Shipments originating in Mexico', '54'],
+        ['MX', 'international', 'UPS Worldwide Saver Shipments originating in Mexico', '65'],
+        ['PL', 'domestic', 'UPS Access Point Economy Polish domestic shipments', '70'],
+        ['PL', 'domestic', 'UPS Today Dedicated Courier Polish domestic shipments', '83'],
+        ['PL', 'domestic', 'UPS Today Express Polish domestic shipments', '85'],
+        ['PL', 'domestic', 'UPS Today Express Saver Polish domestic shipments', '86'],
+        ['PL', 'domestic', 'UPS Today Standard Polish domestic shipments', '82'],
+        ['PL', 'international', 'UPS Expedited Shipments originating in Poland', '08'],
+        ['PL', 'international', 'UPS Express Shipments originating in Poland', '07'],
+        ['PL', 'international', 'UPS Express Plus Shipments originating in Poland', '54'],
+        ['PL', 'international', 'UPS Express Saver Shipments originating in Poland', '65'],
+        ['PL', 'international', 'UPS Standard Shipments originating in Poland', '11'],
+        ['PR', 'domestic', 'UPS 2nd Day Air', '02'],
+        ['PR', 'domestic', 'UPS Ground', '03'],
+        ['PR', 'domestic', 'UPS Next Day Air', '01'],
+        ['PR', 'domestic', 'UPS Next Day Air Early', '14'],
+        ['PR', 'international', 'UPS Worldwide Expedited', '08'],
+        ['PR', 'international', 'UPS Worldwide Express', '07'],
+        ['PR', 'international', 'UPS Worldwide Express Plus', '54'],
+        ['PR', 'international', 'UPS Worldwide Saver', '65'],
+        ['DE', 'domestic', 'UPS Express 12:00 German domestic shipments' '74'],
+        ['OTHER', 'domestic','UPS Express', '07'],
+        ['OTHER', 'domestic','UPS Standard', '11'],
+        ['OTHER', 'international','UPS Worldwide Expedited', '08'],
+        ['OTHER', 'international','UPS Worldwide Express Plus', '54'],
+        ['OTHER', 'international','UPS Worldwide Saver', '65'],
+        ['ALL', 'UPS Worldwide Express Freight' '96'],
```

Comment:
This array has less elements, than the other elements. Also it's missing a comma => SyntaxError. Shouldn't tests have caught this?